ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/CommandParser.java

```diff
@@ -176,6 +186,11 @@ private static SqlCommand transformToSqlCommand(final String sql) {
         return new SqlConnectorStatement(sql);
       case STATEMENT:
         return new SqlStatement(sql);
+      case SET_PROPERTY:
+        final List<String> strings = getStringsInCommand(sql);
+        return new SqlPropertyCommand(sql, true, strings.get(0), strings.get(1));
```

Comment:
I think we should be stricter about verifying what the statement looks like here. Currently, we allow malformed statements such as `SET 'string1' 'string2' 'string3';` which has no actual meaning. Because `SET` and `UNSET` statements are so simple, I think we can use regexes here. The only edge case I can think of that we'll want to check for is whether we require whitespace after `SET`/`UNSET` or not (I don't remember off the top of my head), i.e., is `SET'prop' = 'val';` a valid statement, according to the ksqlDB parser?